helm/superset/values.yaml

```diff
@@ -165,6 +165,25 @@ supersetWorker:
             name: '{{ tpl .Values.envFromSecret . }}'
       command: [ "/bin/sh", "-c", "until nc -zv $DB_HOST $DB_PORT -w1; do echo 'waiting for db'; sleep 1; done" ]
 
+##
+## Superset beat configuration (to trigger scheduled jobs like reports)
+supersetCeleryBeat:
+  # This is only required if you intend to use alerts and reports
+  enabled: false
+  command:
+    - "/bin/sh"
+    - "-c"
+    - ". {{ .Values.configMountPath }}/superset_bootstrap.sh; celery beat --app=superset.tasks.celery_app:app --pidfile /tmp/celerybeat.pid --schedule /tmp/celerybeat-schedule"
```

Comment:
There is no trace of flower in the `master` chart in fact :open_mouth: To add it, we should do something really similar to this PR, plus a service dedicated to flower.
I think this is out of the scope of this PR, my team will switch back to the `master` charts, and we will open PR to add what's missing, but not sure when this will be. :confused: 
If you have some time to do it, I would gladly help!

Reply:
Agreed, different PR... and probably not high prio...

Reply:
Agree, but flower by default does not support auth, and it exposes dangerous functionality and sensitive info. May be a bit out of scope